walletnode/api/services/sense.go

```diff
@@ -115,6 +119,67 @@ func (service *Sense) ActionDetails(ctx context.Context, p *sense.ActionDetailsP
 	return res, nil
 }
 
+// StartProcessing - Starts a processing image task
+func (service *Sense) StartProcessing(ctx context.Context, p *sense.StartProcessingPayload) (res *sense.StartProcessingResult, err error) {
+	ticket := &senseregister.Request{
+		BurnTxID:              p.BurnTxid,
+		AppPastelID:           p.AppPastelID,
+		AppPastelIDPassphrase: p.AppPastelidPassphrase,
+	}
+
+	// get image filename from storage based on image_id
+	filename, err := service.db.Get(p.ImageID)
+	if err != nil {
+		return nil, sense.MakeInternalServerError(err)
+	}
+
+	// get image data from storage
+	file, err := service.register.Storage.File(string(filename))
+	if err != nil {
+		return nil, sense.MakeInternalServerError(err)
+	}
+	ticket.Image = file
+
+	// Create task
+	taskID := service.register.AddTask(ticket)
```

Comment:
first, add task.
and the task will do all the stuff for you and provide status updates. 
again, see artwork register.